src/main/java/com/jalasoft/webservice/model/VideoConverterCriteria.java

```diff
@@ -0,0 +1,107 @@
+package com.jalasoft.webservice.model;
```

Comment:
This class does not have copyright.

Reply:
Thank you for realizing about it because now I am doing the necessary correction.